test/mochitest/browser_dbg-quick-open.js

```diff
@@ -47,10 +47,13 @@ function resultCount(dbg) {
   return findAllElements(dbg, "resultItems").length;
 }
 
-function quickOpen(dbg, query, shortcut = "quickOpen") {
+async function quickOpen(dbg, query, shortcut = "quickOpen") {
   pressKey(dbg, shortcut);
   assertEnabled(dbg);
+
   query !== "" && type(dbg, query);
+
+  await waitForTime(150);
```

Comment:
this is fine, but it would be better if we could wait for something UI related like what we do above with resultCount.

We have several helpers for waiting for UI elements. Again, this is totally fine to ignore